README.md

```diff
@@ -1,21 +1,124 @@
+* Note glide is controllable with the panel's button.
+* Reverb option, with feedback, shimmer, and envelope follower taming infinite feedback as in above mode.
+
+**Filter** Dual multimode stereo filter based on the Mutable Instruments Blades. Provides low pass, band pass, high pass.
+* Left and Right cutoff, mode, resonance independently controllable. Can behave as a stereo filter or two mono filters. 
+* Lower-right knob controls routing, smoothly changing clockwise from parallel to series
+
+**Lo-Fi** Tape-like effect ranging from subtle, saturated tape wobble to brittle, fragile sound from almost any
+input source. Inspired by the Shallow Water pedal.
+* Optional reverb and Chorus
+* Low Pass Gate with knob-controllable cutoff frequency
+* Knob-controllable envelope follower modulation of Low Pass Gate
+* Tape delay effect with wobble rate and depth
+* If LPG cutoff is fully CCW, subtle high Pass filter and saturation added
+
+**Micro Looper** A clockable dual glitch looper with knob-controllable play speed and repetitions. Panel is divided
+into independently looped left and right halves. Like Qu-Bit Data Bender, this mode shines with modulation!
+* Clock input is required, fourth input jack on row 2
+* Buffer freeze via knob
+
+**Delay** A clockable dual/stereo delay with independently controllable channels. Sound is similar to
+the Mimeophon. Unlike many delays, can tolerate clock changes without pitching the audio.
+* Clock input at fourth input jack on row 2
+* Feedback can be tamed with user-controllable filter cutoff
+* Optional reverb
+* Buffer freeze option via knob
+
+**Spectra** Spectral resynthesis of input audio. When a trigger is received, up to six
+active oscillators tune to harmonics of input audio.
+* Resynthesis of incoming audio is triggered at falling edge of trigger/gate input
+at fourth input jack on row 2
+* Pitch controllable at second input jack on row 2
+* Optional reverb
+
+Sounds beautiful with a multiple or division of clock as trigger.
+
+Try *demonizing* speech as the audio input by sending a fast trigger in with Analysis Speed knob turned
+clockwise
+
+**Spectrings** Similar to Spectra, but uses two string voices from Mutable Instruments Rings/Plaits.
+* Resynthesis of incoming audio is triggered at falling edge of trigger/gate input
+  at fourth input jack on row 2.
+* Optional reverb
+
+## Acknowledgements
 
-## Description
-Here's the code for MultiVersio for the Desmodus Versio platform. Build using Visual Studio Code.
 Thanks to Emilie Gilliet (spectrings and many filters are derived from her wonderful work).
-Thanks to the Daisy developers for their lovely platform. And thanks to Noise Engineering for their awesome products!
+Thanks to the Daisy developers at Electrosmith for their lovely platform. And thanks to Noise Engineering for their awesome products!
 
-## More info at:
-https://www.modwiggler.com/forum/viewtopic.php?t=249058
+## Install
 
+1. Download pre-built firmware from the releases page.
```

Comment:
I put this in the doc here as I think it would be pretty easy to use GitHub's release artifact system, even manually, to upload the .bin file as a versioned Release. This is doable all from the GitHub website and wouldn't require anyone to create a GitHub account to download the .bin file and it's a bit cleaner, perhaps, than Dropbox.